src/main/java/com/CezaryZal/manager/creator/NewUser.java

```diff
@@ -0,0 +1,27 @@
+package com.CezaryZal.manager.creator;
+
+import com.CezaryZal.entity.health.calendar.InputUser;
+import com.CezaryZal.entity.health.calendar.UserAuthentication;
+import com.CezaryZal.manager.health.calendar.service.UserHcAuthService;
+import com.CezaryZal.manager.modifier.entity.Converter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class NewUser {
```

Comment:
Co to? jakiś kolejny sposób na tworzenie usera?

Reply:
Kontynuując powyższe, gdy mamy id użytkownika z app HC, przypisujemy go do obiektu, który zostanie zapisany w db, tabeli user_hc.